snowexsql/tables/layer_data.py

```diff
@@ -40,22 +42,25 @@ class LayerData(SingleLocationData, Measurement, Base):
     value = Column(String(50))
     flags = Column(String(20))
 
-    # Link the instrument id with a foreign key
-    instrument_id = Column(
-        Integer, ForeignKey('public.instruments.id'), index=True
-    )
-    # Link the Instrument class
-    instrument = relationship('Instrument')
-
     # Link the site id with a foreign key
     site_id = Column(
         Integer, ForeignKey('public.sites.id'), index=True
     )
     # Link the Site class
-    site = relationship('Site')
+    site = relationship('Site', lazy='joined')
 
     # id is a mapped column for many-to-many with observers
     id: Mapped[int] = mapped_column(primary_key=True)
     observers: Mapped[List[Observer]] = relationship(
         secondary=LayerObservers.__table__
     )
+
+    # # Rendered columns
+    # # Use column_property to reference Site's date column
+    # # This makes querying easier
+    # # date = column_property(
+    # #     select(Site.date).where(Site.id == site_id).scalar_subquery()
+    # # )
+    # date = column_property(
+    #     select(Site.date).where(Site.id == site_id).correlate(Site).scalar_subquery()
+    # )
```

Comment:
What's the intention to keep this around?

Reply:
There isn't one. I was trying to do something clever that didn't work out haha